templates/beta/contracts/BetaTemplateBase.sol

```diff
@@ -42,43 +52,118 @@ contract BetaTemplateBase {
         appIds = _appIds;
     }
 
-    function createDAO(string name, MiniMeToken token, address[] holders, uint256[] stakes, uint256 _maxTokens) internal returns (Voting) {
+    function latestVersionAppBase(bytes32 appId) public view returns (address base) {
+        Repo repo = Repo(PublicResolver(ens().resolver(appId)).addr(appId));
+        (,base,) = repo.getLatest();
+
+        return base;
```

Comment:
Don't need the return here; base is automatically returned.

Reply:
Well, I just copy-pasted it from below, changing the order of the functions, but I can change that too.

Reply:
I think explicit returns are a good idea besides one liners :)

Reply:
Ok, reverting last commit then ;-)